bazel/emscripten_toolchain/link_wrapper.py

```diff
@@ -153,15 +153,11 @@
         wasm_base,
         '--add-section=external_debug_info=debugsection.tmp'])
 
-# If we have more than one output file then create tarball
-if len(files) > 1:
+# If we have one or more output files then create a tarball
+if len(files) >= 1:
```

Comment:
Can we just drop this `if/else` completely.  If you a worried that would loose coverage perhaps just `assert(len(files))` here?

Reply:
OK, but then at least re-write it to use early-return style and avoid the indentation:

```
if not len(files):
  print('emcc.py did not appear to output any known files!')
  sys.exit(1)

.. the rest here without any indentation
```